libs/eo/metering-points/shell/src/lib/eo-metering-point-table.component.ts

```diff
@@ -84,16 +98,22 @@ import { EoMeteringPoint, EoMeteringPointsStore } from './eo-metering-points.sto
         <mat-header-cell *matHeaderCellDef mat-sort-header>Granular Certificates</mat-header-cell>
         <mat-cell *matCellDef="let element">
           <ng-container *ngIf="element.type === 'production'">
-            <span *ngIf="element.contract">Active</span>
-            <ng-container *ngIf="!element.contract">
-              <a class="link" (click)="createContract(element.gsrn)"> Enable </a>
-            </ng-container>
-          </ng-container></mat-cell
-        >
+            <watt-badge *ngIf="element.contract" type="success">Activated</watt-badge>
+            <button
+              *ngIf="!element.contract"
+              [disabled]="(loading$ | async) === true"
+              class="link"
+              (click)="createContract(element.gsrn)"
+            >
+              Activate
+              <watt-spinner *ngIf="(loading$ | async) === true" [diameter]="16"></watt-spinner>
```

Comment:
Using this approach means that when the user clicks "Activate" for a single metering point, then all "Activate" buttons for all other metering points will start spinning and the buttons will be disabled.
Is it possible to change this, so it is only the "Activate" button for that row/metering point that is spinning and disabled? That would also mean, that the other "Activiate" buttons can be clicked; this can result in multiple concurrent calls, but that is not a problem for the API.

Reply:
@duizer yeah sure :) I was not sure if the API could handle it or not, but if it's fine, I will go ahead and make it possible to click more buttons at once

Reply:
Changed it so they can now load seperately